### PR TITLE
Add backup/restore scripts for database

### PR DESCRIPTION
**Pull request by martyngigg**
Adds scripts to easily export and import the database. Restore is particularly useful to test a local copy with a copy of the production database.

### CODE REVIEW

File: bin/backup.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+SCRIPTPATH=$(cd "$(dirname "$0")"; pwd -P)
+SOURCE_DIR=$(cd "$SCRIPTPATH" && cd .. && pwd -P)
+
+source ${SOURCE_DIR}/.env
+
+PROJECT_NAME=reports
+PG_DOCKER_NAME=$PROJECT_NAME"_postgres_1"
+
+docker exec -t $PG_DOCKER_NAME pg_dumpall -c -U ${DB_USER} | gzip > $1
```
Comment (DavidFair) [inline bin/backup.sh:18]: We need to stop the instance to have a consistent backup
We had a corrupt Django table (thankfully an ORM cache) from a previous migration where we took a backup of a live instance. I wonder if there's a way to do this from Django instead so we can backup live?

Comment (martyngigg) [reply bin/backup.sh:18]: It looks to add a set of commands to `manage.py` so we can just run those...

Comment (martyngigg) [reply bin/backup.sh:18]: I'll see if I can get this going.

**Comment by martyngigg** [comment]
Closing this as obselete as the method won't work.

**Comment by martyngigg** [comment]
#27 is open to add this as a command as part of the django management.

**Pull request closed without merging**